guides/assets/javascripts/guides.js

```diff
@@ -52,6 +52,7 @@
     backToTop.addEventListener("click", function(e) {
       e.preventDefault();
       window.scrollTo({ top: 0, behavior: "smooth" });
+      resetNavPosition();
```

Comment:
More of a side note: even though we're doing some reset here once the button is clicked, the scrolling still causes the observer logic to trigger as it's scrolling up, which is a bit weird but not sure if it's anything to worry about, probably not.

Reply:
Yeah, I didn't know what to do about that. I thought the observer logic would take care of resetting the position of the navigation bar but some pages didn't reset all the way to the top without it.